### PR TITLE
ENDOC-551 7.1 release notes

### PR DESCRIPTION
**Pull request by Lyd1aCla1r3**
- current as of 2022-08-22 7PM PST
- WIP
- all bolded terms/phrasing in question

### CODE REVIEW

File: vuepress/docs/next/docs/releases/README.md [hunk 1]
```diff
@@ -1,7 +1,147 @@
...
+- **Post-initialization** phase
+- AppBuilder now extensible through bundles (override of core AppBuilder components and addition of EPC extensions)
+- AppBuilder main menu converted to a overridable bundle with **menu MFE and BFF**
+- Bundles, widgets and pages now created within the scope of a specific bundle (identified **by** using the **bundle-id**)
+- Performance optimizations
+
+### Compatibility
+
+**check**
+* [Entando 7.1 Compatibility](https://entando.com/entando-de-app/cmsresources/cms/documents/Entando_7.0_Compatibility.pdf)
```
Comment (nshaw) [inline vuepress/docs/next/docs/releases/README.md:20]: Add a note somewhere that we need to point this to 7.1 when we have it?

**Review by nshaw — APPROVED** [review]

File: vuepress/docs/next/docs/releases/README.md [hunk 2]
```diff
@@ -1,7 +1,139 @@
...
+- Fixed CVE-2022-22968 (spring-context)
+- Fixed CVE-2022-26336 (apache poi-scratchpad, DoS)
+- Fixed CVE-2022-30126 (apache tika, DoS)
+- Fixed CVE-2022-30973 (apache tika, DoS)
+- Fixed CVE-2022-22976 (spring-security)
+- Fixed CVE-2022-22978 (spring-security-web, Authorization Bypass)
+- Fixed CVE-2022-33879 (apache tika, DoS)
+- Fixed CVE-2022-24823 (apache netty, Information Exposure)
+
+### **AdminConsole:**
```
Comment (jyunmitch) [inline vuepress/docs/next/docs/releases/README.md:40]: is there a reason why this is bolded with a colon?

File: vuepress/docs/next/docs/releases/README.md [hunk 3]
```diff
@@ -1,7 +1,139 @@
...
+   - Provides App Builder with alternative core components
+   - Introduces new screens accessible from a menu dedicated to EPCs
+- The same SEO-friendly code can be used for different languages on the same page
+- CSS/text files can be viewed or edited in the App Builder file browser
+- Renamed Entando Component Repository (ECR) to Local Hub
+- Improved error handling when uninstalling a bundle
+- Fixed the clone function for UX fragments
+- Minor adjustments to support docker-based bundles 
+- Minor fixes related to pagination and usability
+- Invoking APIs now sends the authentication header (instead of explicitly sending authentication cookies)
```
Comment (jyunmitch) [inline vuepress/docs/next/docs/releases/README.md:57]: parenthesis not needed here

File: vuepress/docs/next/docs/releases/README.md [hunk 4]
```diff
@@ -1,7 +1,139 @@
...
+### AppEngine
+
+- Enabled Keycloak authentication for additional core APIs
+- Optimized startup procedure in cases where an automatic backup restore is implied
+- All APIs now authenticate exclusively through HTTP headers and do not require cookies
+- Fixed session invalidation issues
+
+### Entando CLI (ent)
+
+- Dropped support for Entando <= 6.3.0 and Kubernetes <= 1.18
```
Comment (jyunmitch) [inline vuepress/docs/next/docs/releases/README.md:69]: Maybe I'm missing something, what does this mean? those arrows confuse things

File: vuepress/docs/next/docs/releases/README.md [hunk 5]
```diff
@@ -1,7 +1,139 @@
...
+- [Entando Bundle CLI](#entando-bundle-cli) (`ent bundle` tool) added to ent
+- `check-env` updated to automatically install additional dependencies from GitHub packages
+- `get-plugin-id` deprecated in favor of `get-plugin-code`
+- Minor `ent pkg` fixes
+- Added [crane](https://github.com/google/go-containerregistry/blob/main/cmd/crane/doc/crane.md) as an external package
+- Added shortcuts for running internal k9s, crane, jq and fzf (e.g. `ent crane`)
+- `ent attach-kubectx` now supports `--current` to attach the current kubectl context (e.g. following an `oc login`)
+- Modified `get-bundle-id` to support docker-based bundles
+- Fixes to `ent help` and logging
```
Comment (jyunmitch) [inline vuepress/docs/next/docs/releases/README.md:78]: logging is strange without an object>> logging management? (from below)

File: vuepress/docs/next/docs/releases/README.md [hunk 6]
```diff
@@ -1,7 +1,139 @@
...
+- Now jq is always required by `check-env` (both runtime and develop)
+- Experimental support for an enhanced selection menu based on fzf (`ent pkg get fzf`, `ent config --set FZF_SELECT true`)
+- Workarounds to mitigate npm bugs in the update process
+- `ent-diag` now extracts additional details
+
+### Entando Bundle CLI
+
+- New CLI module "ent bundle" to manage docker-based bundles
+- Support for the following workflows:
+   - From scratch: User can create a new bundle project and then add components from existing projects or using external tools
```
Comment (jyunmitch) [inline vuepress/docs/next/docs/releases/README.md:92]: using external tools >> use external 
to match add components and use external...

File: vuepress/docs/next/docs/releases/README.md [hunk 7]
```diff
@@ -1,7 +1,139 @@
...
+- A single project file "entando.json" to describe the entire bundle structure, customizations and parameters
+- Support for App Builder EPC extensions
+- Support for all properties of git-based bundle YAML descriptors
+- Support for all properties of docker-based bundles
+- Support for service discovery via the [API claims mechanism](../getting-started/ent-api.md)
+- Source-level support for bundle thumbnails
+- Capability to customize the build, run and packaging commands
+- Custom resource descriptor generation for the current bundle and external Docker repositories
+- Improved log management during parallel builds
+- Improved local testing capabilities and auxiliary services management based on docker-compose
```
Comment (jyunmitch) [inline vuepress/docs/next/docs/releases/README.md:106]: Docker Compose

File: vuepress/docs/next/docs/releases/README.md [hunk 8]
```diff
@@ -1,7 +1,139 @@
...
+- Fixed performance issues caused by unnecessary reprocessing of all Content Types
+
+### Infrastructure / Local Hub
+
+- Entando Component Repository renamed to Local Hub
+- Support for docker-based bundles
+- File structure deployed to AppEngine is redesigned with improved scoping
+- Added new MFE widget capabilities:
+   - Support for service discovery driven by the [API claims mechanism](../getting-started/ent-api.md)
+   - Automatic generation of the widget configUi
```
Comment (jyunmitch) [inline vuepress/docs/next/docs/releases/README.md:120]: >>configUI

File: vuepress/docs/next/docs/releases/README.md [hunk 9]
```diff
@@ -1,7 +1,139 @@
...
+   - Widgets can receive environmental and system information through a config object
+   - Widgets can now access their own static assets
+   - Support for explicitly parameterized widgets
+   - Asset URLs are now properly scoped
+- Improvements to EntandoPlugin ingress management:
+   - Canonical ingress path: Determined by the infrastructure and used extensively, from internal communication to service discovery
+   - Customized ingress path: An alternative path that can be used by 3rd parties
+- Retrocompatible changes to the EntandoPlugin CRD to store both the customized and canonical paths
```
Comment (jyunmitch) [inline vuepress/docs/next/docs/releases/README.md:128]: Can we use Backward compatible here? More common usage

File: vuepress/docs/next/docs/releases/README.md [hunk 10]
```diff
@@ -1,7 +1,139 @@
...
-## New Features and Major Fixes
-This information will be provided when the next version of Entando is released.
+Entando 7.1.0 is an important feature release that introduces major changes to the bundle subsystem. It supports a brand new create phase, bundle service discovery and App Builder extensibility via Entando Packaged Capabilities (EPCs).
+
+## ABSTRACT
+
+- Security fixes
+- Node-based CLI module to manage bundle projects
+- Decoupled MFE ⇒ MS communication via service discovery ([API claims](../getting-started/ent-api.md))
+- New docker-based bundles (bundles entirely distributed via Docker/OCI images)
```
[the remark this answers is not in the document]
Comment (nshaw) [reply vuepress/docs/next/docs/releases/README.md:10]: Docker image is synonymous with OCI image. I think this language works.

File: vuepress/docs/next/docs/releases/README.md (hunk 4 above)
Comment (nshaw) [reply vuepress/docs/next/docs/releases/README.md:69]: Less than or equal to 6.3.0, etc. Math-y.

File: vuepress/docs/next/docs/releases/README.md (hunk 2 above)
Comment (Lyd1aCla1r3) [reply vuepress/docs/next/docs/releases/README.md:40]: yes lol i wanted to confirm it should be "Admin Console" but missed it during our triage

File: vuepress/docs/next/docs/releases/README.md (hunk 5 above)
Comment (Lyd1aCla1r3) [reply vuepress/docs/next/docs/releases/README.md:78]: ? logging works well by itself. logging is a thing.

File: vuepress/docs/next/docs/releases/README.md (hunk 6 above)
Comment (Lyd1aCla1r3) [reply vuepress/docs/next/docs/releases/README.md:92]: disagree. it reads "add components...using external tools"

File: vuepress/docs/next/docs/releases/README.md (hunk 7 above)
Comment (Lyd1aCla1r3) [reply vuepress/docs/next/docs/releases/README.md:106]: docker-compose

File: vuepress/docs/next/docs/releases/README.md (hunk 8 above)
Comment (Lyd1aCla1r3) [reply vuepress/docs/next/docs/releases/README.md:120]: @nshaw how is this formatted in code?

Comment (nshaw) [reply vuepress/docs/next/docs/releases/README.md:120]: configUi - https://github.com/entando-samples/e70-react-with-config/blob/main/ui/widgets/widgets-dir/et-widget/bundle/et-widget-descriptor.yaml

File: vuepress/docs/next/docs/releases/README.md (hunk 4 above)
Comment (jyunmitch) [reply vuepress/docs/next/docs/releases/README.md:69]: we can't do ≤ ?

File: vuepress/docs/next/docs/releases/README.md (hunk 7 above)
Comment (jyunmitch) [reply vuepress/docs/next/docs/releases/README.md:106]: to clarify, when is it docker-compose?

File: vuepress/docs/next/docs/releases/README.md (hunk 2 above)
Comment (nshaw) [reply vuepress/docs/next/docs/releases/README.md:40]: Lose this section, combine with App Builder.

File: vuepress/docs/next/docs/releases/README.md [hunk 11]
```diff
@@ -1,7 +1,139 @@
...
+- Fixed CVE-2022-30973 (apache tika, DoS)
+- Fixed CVE-2022-22976 (spring-security)
+- Fixed CVE-2022-22978 (spring-security-web, Authorization Bypass)
+- Fixed CVE-2022-33879 (apache tika, DoS)
+- Fixed CVE-2022-24823 (apache netty, Information Exposure)
+
+### **AdminConsole:**
+
+- Improved handling of special chars in page titles
+- Improved visual and functional integration with new AppBuilder EPC menu
```
Comment (nshaw) [inline vuepress/docs/next/docs/releases/README.md:43]: Lose this line, internal implementation detail.

File: vuepress/docs/next/docs/releases/README.md (hunk 8 above)
Comment (nshaw) [reply vuepress/docs/next/docs/releases/README.md:120]: "Automatic generation of the widget `configUi` integration code"

File: vuepress/docs/next/docs/releases/README.md (hunk 4 above)
Comment (nshaw) [reply vuepress/docs/next/docs/releases/README.md:69]: Hah, good idea. Correction, old-school-developer-math-y. ;)

**Review by jyunmitch — APPROVED** [review]

**Pull request merged**